bin-dev-scripts/standard-version-release.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash -e
```

Comment:
generally, use `#!/usr/bin/env sh` and `set -o nounset -o pipefail -o errexit` . And prefer not to use bash specific syntax in the scripts. I don't see any bash specific syntax, so thats good. Why `sh` and not `bash`, cuz bash is not guaranteed to always be there. `sh` represents whatever is the default shell for the user running the script. I prefer the explicit `set` commands as opposed to the `-e` commandline option for bash.

Reply:
Not a blocker, just mentioning it.

Reply:
Okay, will read on this and update the script.